PGM/src/main/java/tc/oc/pgm/polls/commands/PollSubCommands.java

```diff
@@ -0,0 +1,186 @@
+import tc.oc.commons.core.formatting.StringUtils;
+import tc.oc.commons.core.restart.RestartManager;
+import tc.oc.pgm.Config;
+import tc.oc.pgm.commands.CommandUtils;
+import tc.oc.pgm.map.PGMMap;
+import tc.oc.pgm.match.MatchManager;
+import tc.oc.pgm.mutation.Mutation;
+import tc.oc.pgm.mutation.MutationQueue;
+import tc.oc.pgm.mutation.command.MutationCommands;
+import tc.oc.pgm.polls.PollBlacklist;
+import tc.oc.pgm.polls.types.PollCustom;
+import tc.oc.pgm.polls.types.PollKick;
+import tc.oc.pgm.polls.PollManager;
+import tc.oc.pgm.polls.types.PollMutation;
+import tc.oc.pgm.polls.types.PollNextMap;
+
+import javax.inject.Inject;
+import java.util.Collection;
+import java.util.List;
+
+public class PollSubCommands {
+
+    private final RestartManager restartManager;
+    private final MutationQueue mutationQueue;
+    private final PollManager pollManager;
+    private final PollCustom.Factory pollCustomFactory;
+    private final PollNextMap.Factory pollMapFactory;
+    private final PollMutation.Factory pollMutationFactory;
+    private final PollKick.Factory pollKickFactory;
+    private final PollBlacklist pollBlacklist;
+    private final UserStore userStore;
+    private final OnlinePlayers onlinePlayers;
+    private final MatchManager matchManager;
+
+    @Inject
+    PollSubCommands(RestartManager restartManager,
+                 MutationQueue mutationQueue,
+                 PollManager pollManager,
+                 PollCustom.Factory pollCustomFactory,
+                 PollNextMap.Factory pollMapFactory,
+                 PollMutation.Factory pollMutationFactory,
+                 PollKick.Factory pollKickFactory,
+                 PollBlacklist pollBlacklist,
+                 UserStore userStore,
+                 OnlinePlayers onlinePlayers,
+                 MatchManager matchManager) {
+        this.restartManager = restartManager;
+        this.mutationQueue = mutationQueue;
+        this.pollManager = pollManager;
+        this.pollCustomFactory = pollCustomFactory;
+        this.pollMapFactory = pollMapFactory;
+        this.pollMutationFactory = pollMutationFactory;
+        this.pollKickFactory = pollKickFactory;
+        this.pollBlacklist = pollBlacklist;
+        this.userStore = userStore;
+        this.onlinePlayers = onlinePlayers;
+        this.matchManager = matchManager;
+    }
+
+
+
+    @Command(
+        aliases = {"kick"},
+        desc = "Start a poll to kick another player.",
+        usage = "[player]",
+        min = 1,
+        max = 1
+    )
+    @CommandPermissions("poll.kick")
+    public void pollKick(CommandContext args, CommandSender sender) throws CommandException {
+        Player initiator = tc.oc.commons.bukkit.commands.CommandUtils.senderToPlayer(sender);
+        Player player = tc.oc.commons.bukkit.commands.CommandUtils.findOnlinePlayer(args, sender, 0);
+
+        if(player.hasPermission("pgm.poll.kick.exempt") && !initiator.hasPermission("pgm.poll.kick.override")) {
```

Comment:
Declare the permission strings as a constant at the top of the class, just so other classes may use it for whatever purposes